pages/profile/read/views/ppl-apply.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Snippet } from '@asl/components';
+import { Button } from '@ukhomeoffice/react-components';
+
+class PplApply extends React.Component {
+  render() {
+    const { allowedActions, estId } = this.props;
+
+    if (!allowedActions.includes('project.apply')) {
+      return null;
+    }
+    return (
+      <div>
+        <p>
+          <form method='POST' action={`/e/${estId}/projects/create`}>
+            <Button className='govuk-button add-margin'>
+              <Snippet>buttons.pplApply</Snippet>
+            </Button>
+          </form>
+        </p>
+      </div>
```

Comment:
I'm not sure there's any need for this wrapper `<div>`

Reply:
Because the other two had a wrapper - i.e https://github.com/UKHomeOffice/asl-pages/blob/444e63ba94ce3729e99776805b176473d08fb81d/pages/profile/read/views/role-apply.jsx#L12 and https://github.com/UKHomeOffice/asl-pages/blob/444e63ba94ce3729e99776805b176473d08fb81d/pages/profile/read/views/pil-apply.jsx#L15 I thought there is a pattern to follow

Reply:
less is more when it comes to HTML - we don't want to be adding <div>'s that don't serve any purpose - this isn't a pattern to follow, the others probably made it in as they were missed when reviewing

Reply:
I am getting rid of all ?